src/containerapp/azext_containerapp/_params.py

```diff
@@ -155,6 +155,7 @@ def load_arguments(self, _):
         c.argument('certificate_file', options_list=['--certificate-file', '-f'], help='The filepath of the .pfx or .pem file')
         c.argument('certificate_name', options_list=['--certificate-name', '-c'], help='Name of the certificate which should be unique within the Container Apps environment.')
         c.argument('certificate_password', options_list=['--password', '-p'], help='The certificate file password')
+        c.argument('prompt', options_list=['--show-prompt'], help='Show prompt to upload an existing certificate.')
```

Comment:
Could we define it as [`action='store_true'`](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-flags)?